specification/specs/common_options/date_math/DateMathExpression.ts

```diff
@@ -1,2 +1,2 @@
 @class_serializer("DateMathExpressionFormatter")
```

Comment:
can not use annotations :(
```suggestion
/*
 * @class_serializer  DateMathExpressionFormatter
 */
```